setup.py

```diff
@@ -24,6 +24,8 @@
         "spark": ["pyspark>=2.3.2"],
         "sqlalchemy": ["sqlalchemy>=1.2"],
         "airflow": ["apache-airflow[s3]>=1.9.0", "boto3>=1.7.3"],
+        "redshift": ["psycopg2>=2.8"],
+        "s3": ["boto3>=1.14"],
```

Comment:
This is unrelated. Why did you add this?

Reply:
While we are at it it might be nice to add common GCP libraries such as:
```
google-cloud==0.34.0
google-cloud-secret-manager==1.0.0
google-cloud-storage==1.29.0
pybigquery==0.4.15
```

Reply:
As I tested the Docker image in "Production settings" I had to add `boto3` to the image.

The set up right now at Earnest is: starting the Great Expectations container through Airflow which publishes docs and validations to S3.

That's why I added the `s3` dep

Reply:
Added the GCP deps here https://github.com/great-expectations/great_expectations/pull/1584/commits/8378c99c1256ff4b346e31dff42a14fb0bfd3eec

Reply:
Sounds reasonable.